DependencyInjection/Configuration.php

```diff
@@ -111,6 +111,7 @@ public function getConfigTreeBuilder()
                                         ->integerNode('port')->defaultValue(6379)->end()
                                         ->scalarNode('host')->defaultValue('localhost')->end()
                                         ->integerNode('database')->isRequired()->end()
+                                        ->integerNode('password')->defaultValue(null)->end()
```

Comment:
> Database was always an integerNode, so no change there

I know, but still, I wonder how this can work at all, don't you?

> I didn't pay attention to the detail when editing the pull request and kept the integerNode on the password key, so apologies there.

No apology needed, it's ok, just make sure it works at least for you in your project.